build/debian-base/Makefile

```diff
@@ -51,6 +51,10 @@ ifeq ($(ARCH),s390x)
 	BASEIMAGE?=s390x/debian:buster-slim
 	QEMUARCH=s390x
 endif
+ifeq ($(ARCH),mips64le)
+	BASEIMAGE?=multiarch/debian-debootstrap:mips64el-buster-slim
```

Comment:
hmm I'm not sure about this aspect

Reply:
[Multiarch](https://hub.docker.com/r/multiarch/qemu-user-static) is an image repository provided by qemu.
multiarch/debian-debootstrap:mips64el-buster-slim was built by qemu using debootstrap. This is its production [script](https://github.com/multiarch/debian-debootstrap/blob/master/update.sh).

Reply:
I see `https://hub.docker.com/u/mips64le` org but don't see anything published under this repo? all of those images mentioned here for other architectures built and well maintained by the docker-library project,

Reply:
@mkumatag Actually the `/mips64le` org is working in progress, maybe these images will be avaliable soon. See https://github.com/docker-library/official-images/issues/6709

Reply:
hope things will settle soon 🤞